gui/tools/updatessl.py

```diff
@@ -27,30 +27,15 @@
 
```

Comment:
This script was used back in 2014. I think its time we retire this. If anyone is upgrading from 9.3 we can ask them to upgrade to 11.2 first.

(We will also need to update ix-update because of this).